### PR TITLE
39 tenfh wlr

### PR DESCRIPTION
**Pull request by lili-ling-msd**

### CODE REVIEW

File: R/pMaxCombo.R [hunk 1]
```diff
@@ -23,12 +23,12 @@ NULL
 #' MaxCombo p-value
 #'
 #' \code{pMaxCombo()} computes p-values for the MaxCombo test
-#' based on output from \code{simtrial::tenFHcorr()}.
```
Comment (LittleBeannie) [inline R/pMaxCombo.R:26]: Hi @lili-ling-msd , could you please revert `wlrcorr()` back as `tenFHcorr()`?

**Review by LittleBeannie — CHANGES_REQUESTED** [review]
Nice catches of all `tenFH`! I only have 1 minor comment which I guess would be easy and quick to fix.

File: R/pMaxCombo.R (hunk 1 above)
Comment (lili-ling-msd) [reply R/pMaxCombo.R:26]: sure

**Review by LittleBeannie — APPROVED** [review]
There are a couple of conflicts since the master is updated with merges. No worries, I already fixed it. Thank you for keeping working on it!

**Pull request merged**